src/app/features/exchange-set/ess-add-single-encs/ess-add-single-encs.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+<div class="addSingleFileSection">
+    <ukho-textinput class="addSingleFileTextbox" label="Select a single ENC" [placeholder]="placeholder"></ukho-textinput>
```

Comment:
Is there a reason why placeholder is populated by a binding but the label property is not? If no, could you please replace the placeholder binding with the literal string

Reply:
Resolved